src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,106 @@
+
+package duke;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+
+public class Parser {
+
+
+    public String startParse() {
+        System.out.println("Hello! I'm Duke What can I do for you?");
+        ArrayList<Task> array = new ArrayList<>(100);
+        int counter = 0;
+        boolean flag = false;
```

Comment:
For naming, perhaps this variable can be named as isXX because it is a boolean value. 🤔